src/middlewared/middlewared/plugins/activedirectory.py

```diff
@@ -1015,6 +1015,13 @@ class Config:
             await self.middleware.call('activedirectory.get_cache')
             if ad['verbose_logging']:
                 self.logger.debug('Successfully started AD service for [%s].', ad['domainname'])
+
+            if smb_ha_mode == "LEGACY":
+                job.set_progress(95, 'starting active directory on standby controller')
+                try:
+                    await self.middleware.call('failover.call_remote', 'activedirectory.start')
```

Comment:
Wont that cause an infinite loop of calls? a node call the remote node, and that remote node call the other remote node again?